src/sort-numbers/built-in-sort-rayane/sort-numbers-rayane.spec.js

```diff
@@ -0,0 +1,99 @@
+import { sortArrayOfNumbers } from './sort-numbers-rayane.js';
+
+describe('sortArrayOfNumbers sorts an array of numbers', () => {
+  describe('sorts positive numbers', () => {
+    it('whole numbers', () => {
+      const received = sortArrayOfNumbers([25, 45, 10, 3, 81]);
+      const expected = [3, 10, 25, 45, 81];
+      expect(received).toEqual(expected);
+    });
+    it('decimal numbers', () => {
+      const received = sortArrayOfNumbers([9.4, 0.01, 4.1, 1.11]);
+      const expected = [0.01, 1.11, 4.1, 9.4];
+      expect(received).toEqual(expected);
+    });
+    it('mixed whole and decimal numbers', () => {
```

Comment:
here, the example only includes decimal numbers, while it's described as mixed whole and decimal